obsei/preprocessor/base_text_cleaner.py

```diff
@@ -0,0 +1,39 @@
+from abc import abstractmethod
+from typing import Any, Dict, List, Optional
+
+from pydantic import BaseModel, PrivateAttr
+
+
+class TextCleanerRequest(BaseModel):
```

Comment:
For time being just use `AnalyzerRequest` for input and output for clean_input function.
There is plan to make input and output more consistent between each node of DAG.

  `AnalyzerRequest` have two major part `processed_text` and `meta`.
`processed_text` is modifiable part hence cleaner, translation and transformation can modify it. And Meta is add only data where each node can add new data only that to in it's own space but can't modify or remove